listener/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/kyma-project/runtime-watcher/listener
 go 1.23.2
 
 // TODO: https://github.com/kyma-project/runtime-watcher/issues/402
-replace golang.org/x/text v0.16.0 => golang.org/x/text v0.19.0
+replace golang.org/x/net v0.26.0 => golang.org/x/net v0.30.0
```

Comment:
to reviewer: the replace for `x/text` directly didn't work. Presumably because `x/net` required this version explicitly. Fixed now by replacing `x/net` instead which bumped its required `x/text` dependency.